docs/content/guides/dagster/recommended-project-structure.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Structuring your Dagster project  | Dagster Docs
+description: This guide talks about Dagster's recommendations on structuring larger projects
+---
+
+# Structuring your Dagster project
+
+<Note>
+  <strong>Got questions about our recommendations, or something to add?</strong>{" "}
+  Join our [GitHub discussion]() to share how you organize your Dagster code.
+</Note>
```

Comment:
```suggestion
<Note>
  <strong>Got questions about our recommendations or something to add?</strong>{" "}
  Join our <a href="https://github.com/dagster-io/dagster/discussions/8972">GitHub discussion</a> to share how you organize your Dagster code.
</Note>
```

Adding the discussion link, along with formatting this as an HTML link. You'll likely need to run `mdx-format` again for this (bleh)